<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ejmdGeneralHintsSubmissionWidget</class>
 <widget class="QWidget" name="ejmdGeneralHintsSubmissionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Time Solve</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="suTimeSolve"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time Submit</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="suTimeSubmit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attempts</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="suAttempts"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Code Lines</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QLineEdit" name="suCodeLines"/>
   </item>
   <item row="4" column="0" rowspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Length</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="suLength"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
